### PR TITLE
RESOLVES #1730 and #2205: 'new GRH script'

### LINKED ISSUE #1730: NOTES - PostPay GRH Non-HRF
Our GRH unit asking if we can build a script that would run for the non-HRF cases.  I have met a few time to go over what was needed and started building one.  I will share with the rest once done, and will be asking for help.

Thanks,

### PR DESCRIPTION
**Pull request by kennyasalee**
BLIP 'Case note script that will screen into non-hrf GRH cases and process GRH post payment.'

### CODE REVIEW

File: Script Files/NOTES/NOTES - GRH - NON-HRF-POSTPAY.vbs [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+  Text 185, 60, 25, 10, "* JOBS"
+  Text 240, 40, 25, 10, "* UNEA"
+  Text 240, 50, 25, 10, "* PBEN"
+  Text 240, 60, 50, 10, "* VNDS"
+  GroupBox 170, 5, 135, 70, "Description:"
+EndDialog
+
+'Second Dialog when all info has been grab from case will be called into fields/variants to be reviewed by worker
+BeginDialog GRH_case_note_dialog, 0, 0, 456, 275, "GRH NON-HRF CASE NOTE dialog"
+  EditBox 80, 5, 365, 15, addr_faci_vnds_status
```
Comment (CDPotter) [inline Script Files/NOTES/NOTES - GRH - NON-HRF-POSTPAY.vbs]: Should we make these mandatory fields? (addr_faci_vnds_status) (postpay_results)

File: Script Files/NOTES/NOTES - GRH - NON-HRF-POSTPAY.vbs [hunk 2]
```diff
@@ -0,0 +1,530 @@
...
+	If ButtonPressed = ELIG_GRH_button then
+		call navigate_to_MAXIS_screen("elig", "GRH_")
+		Transmit
+		Transmit
+		Transmit
+	End If
+	'Goes to MONY VNDS screen using the most active faci vnd number on case... If EMReadScreen does not read any FACI pnls. MsgBox there are no faci pnls.
+	If ButtonPressed = VNDS_button then
+		If faci_pnls = "0" then
+			MsgBox "There Are No Faci pnls"
```
Comment (CDPotter) [inline Script Files/NOTES/NOTES - GRH - NON-HRF-POSTPAY.vbs]: Please change any instance of displaying pnl (variables are fine) to the full word for end user readability

File: Script Files/NOTES/NOTES - GRH - NON-HRF-POSTPAY.vbs [hunk 3]
```diff
@@ -0,0 +1,530 @@
...
+'declares variant earnincome_status for dialog/case notes
+earnincome_status = jobs_status & ", " & busi_status
+
+'checks UNEA and types of UNEA'
+Call MAXIS_case_number_finder(MAXIS_case_number)
+Call navigate_to_MAXIS_screen ("STAT", "UNEA")
+EMReadScreen unea_pnls, 1, 2, 78		'counts how many active unea pnls
+Dim unea_list()					'Dims variable to make an array list of existing UNEA pnls
+ReDim unea_list(unea_pnls)
+u = 1
```
Comment (CDPotter) [inline Script Files/NOTES/NOTES - GRH - NON-HRF-POSTPAY.vbs]: What are the U and R used for? Could you comment that into the code?

File: Script Files/NOTES/NOTES - MAIN MENU.vbs [hunk 4]
```diff
@@ -335,6 +335,13 @@ script_array_G_to_L(script_num).description				= "Template for GRH HRFs. Case mu
 script_num = script_num + 1								'Increment by one
 ReDim Preserve script_array_G_to_L(script_num)			'Resets the array to add one more element to it
 Set script_array_G_to_L(script_num) = new script		'Set this array element to be a new script. Script details below...
+script_array_G_to_L(script_num).script_name 			= "GRH - NON-HRF "
```
Comment (CDPotter) [inline Script Files/NOTES/NOTES - MAIN MENU.vbs]: Could we include Postpay in the button name? Might help people who don't like reading descriptions try it.

**Comment by CDPotter** [comment]
Looking good, please fix the few comments I had and I think we're in business!

**Comment by CDPotter** [comment]
@kennyasalee any chance to get these fixes implemented?

**Comment by kennyasalee** [comment]
ok im going to work on these today. will try to get it out by Friday! Thanks @CDPotter !

**Comment by kennyasalee** [comment]
I believe (post_pay_results) should not be mandatory. case may not be an active post pay/non hrf post pay case anymore. This is however, just a tool should the worker choose to capture more info into case note. this can be left blink since (actions_taken) already clears the worker's action.

**Comment by CDPotter** [comment]
@kennyasalee I think I'm going to close this one since it's been sitting out for so long and we can look at it next Monday

**Comment by kennyasalee** [comment]
ok thanks @CDPotter !

**Pull request closed without merging**